tests/gpu/GPUUnitTest.cpp

```diff
@@ -501,13 +504,37 @@ namespace
     }
 };
 
+#if __APPLE__
```

Comment:
From line 507 to line 525, the code can loop on arguments for all platforms, but only macOS has an argument to catch